tfx/components/example_gen/base_example_gen_executor.py

```diff
@@ -176,15 +172,15 @@ def GenerateExamplesByBeam(
       pipeline: beam.Pipeline,
       exec_properties: Dict[Text, Any],
   ) -> Dict[Text, beam.pvalue.PCollection]:
-    """Converts input source to TF example splits based on configs.
+    """Converts input source to TF record splits based on configs.
```

Comment:
for the usage for `TF record`, please see the comment below.

Reply:
i replace TF record with 'serialized record', is this ok?